README.md

```diff
@@ -1,37 +1,44 @@
+This repo stores the data that
 
-People can update their listing by submitting a PR for approval. After a PR is merged, the CNCF site will reflect the update within 10 min. The listings are:
+1. populates various people listing pages on the cncf.io website
+2. grants access to repositories in the CNCF GitHub Org
+
+## Update your listing(s) on the cncf.io website pages with people.json
+
+You can update your listing in people.json by submitting a pull request for approval. After the PR is merged, the CNCF site will reflect the update within 10 min.
+
+The listings include:
 
 - [Ambassadors](https://www.cncf.io/people/ambassadors/)
 - [Governing Board](https://www.cncf.io/people/governing-board/)
 - [Staff](https://www.cncf.io/people/staff/)
 - [Technical Oversight Committee](https://www.cncf.io/people/technical-oversight-committee/)
 
-CNCF community members can request access to any repo by submitting a PR to change the config.yaml file in this repo.
+## Requesting access to CNCF repositories using CLOWarden and config.yaml
 
-## Listing Formats
+The CNCF has centralized access control for repos in this org using a application called CLOWarden [src](https://github.com/cncf/clowarden) [site](https://clowarden.io/audit/).
 
-### config.yaml configures CNCF org repository access
+Submitting Pull request to change config.yaml is how you request repo access.
 
-Use config.yaml here to control repo-level access to your CNCF org repository.
+Your submitted changes will be validated on the Pull Request by CLOWarden. If there are any problems with your request, you will receive clear instructions from CLOWarden on how to fix them.
 
-config.yaml documents the people, and teams of people, who are granted access to GitHub repositories in the CNCF org.
+The rules contained in config.yaml are the single source of truth that document who has access to repos in the CNCF Org on Github and represent the desired state that is read by CLOWarden. 
 
-DO NOT USE settings.yml at the repo-level to control permission; if you do, [Sheriff](#notes) will overwite permissions described in the repo-level settings.yml with the settings described here in config.yaml.
+If you grant access to a repo by any other means (manually using the GitHub web app, updating settings.yml in a repos .github directory) and the access granted is not already described in the config.yaml file here, then CLOWarden will revert the access granted each time it checks the config.yaml file.
 
-A person's GitHub profile is used to grant access to a repository or define membership of a team.
-
-Adding an entry to repositories allow you to describe who has access to your repoistory.
+There are two ways to grant repo access to a community member in config.yaml. You can add their Github Profile to a repo entry or a named team. If you add them to a repo entry then the user will be granted the stated permission level to just that repo.
```

Comment:
I would use GitHub `username` instead of profile, here and in the sample config snippet.

I'd also suggest highlighting that the `username` is case sensitive, so it should always be typed exactly as it appears in the user's profile page. Similarly, I would explain that teams maintainers must belong to the organization before being added to the teams.

> [!IMPORTANT]
> The points above have bitten us a few times, so I would use an `important` block like this one to make them stand out.

Reply:
@tegioz Is it maintainers only that need to invited to the org?